docs/getting-started/getting-help/prophecy-details.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Send Prophecy details to Support
+id: prophecy-details
+sidebar_position: 1
+description: How to download logs and send for support
+sidebar_label: Send info from Prophecy
+tags: []
+---
+
+Gather logs from your Prophecy environment and send them to the support team via the [Support Portal](https://prophecy.zendesk.com/).
+
+<table>
+  <thead>
+    <tr>
+      <th>Log to Send</th>
+      <th>        Example</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Prophecy <a href="https://docs.prophecy.io/architecture/self-hosted/download-logs/#enable-prophecy-downloads-logs">System logs.</a></td>
```

Comment:
@kathweinschenkprophecy do you know a way to use a relative path for this type of link? I worry about links breaking when we re-arrange our docs.

Reply:
@kathweinschenkprophecy I found this and will try now: https://github.com/facebook/docusaurus/issues/6810